AirCasting/Map/MapNotesViewModel.swift

```diff
@@ -0,0 +1,30 @@
+// Created by Lunar on 05/01/2022.
+//
+
+import Foundation
+import CoreLocation
+import UIKit
+
+struct MapNote {
+    let id: Int
+    let location: CLLocationCoordinate2D
+    let markerImage: UIImage
+}
+
+class MapNotesViewModel: ObservableObject {
```

Comment:
tapping a note should also be handlede here, but for that we'd need some form of a router class/block here so maybe let's skip it for now since those mapviews are really a mess and need refactoring

Reply:
Should I make a task for a future: "make a router" or sth? Or we can just skip at all?